src/views/common/WithdrawDash.tsx

```diff
@@ -0,0 +1,198 @@
+					<HStack pr={{ base: "0rem", md: "1rem" }}>
+						<Text fontSize={{ base: fontSizes.sm, md: fontSizes.md }}  >Health factor</Text>
+					</HStack>
+					<HStack pr={{ base: "0rem", md: "1rem" }} textAlign="center" w="100%">
+						<ColoredText minW={{base:'30px',md:"100%"}}> {healthFactor?.toUnsafeFloat().toLocaleString() ?? "-"}</ColoredText>
+						<ModalIcon position="relative" top="0" right="0" onOpen={() => { }} />
+					</HStack>
+				</Flex>
+				<Flex spacing={spacings.md} mr={{ base: "0rem", md: "1rem" }} alignItems="center" justifyContent="flex-start" flexDirection="column">
+					<HStack pr={{ base: "0rem", md: "1rem" }}>
+						<Text fontSize={{ base: fontSizes.sm, md: fontSizes.md }}>{isSmallerThan900 ? "Current LTV" : "Current LTV"}</Text>
+					</HStack>
+					<HStack pr={{ base: "0rem", md: "1rem" }} textAlign="center">
+						<Text fontSize={{ base: fontSizes.md, md: fontSizes.lg, lg: fontSizes.xl }} fontWeight="bold" minW={{base:'30px',md:"100%"}}>
+							{currentLtv ? (currentLtv.toUnsafeFloat() * 100).toLocaleString().slice(0, 6) : "-"} %
+						</Text>
+						<ModalIcon position="relative" top="0" right="0" onOpen={() => { }} />
+					</HStack>
+				</Flex>
+				<Flex spacing={spacings.md} mr={{ base: "0rem", md: "1rem" }} alignItems="center" justifyContent="flex-start" flexDirection="column">
+					<HStack px={{ base: "0rem", md: "1rem" }}>
+						<Text fontSize={{ base: fontSizes.sm, md: fontSizes.md }}>{isSmallerThan900 ? "Collateral" : "Collateral Composition"}</Text>
+					</HStack>
+					<Popover trigger='hover' >
+						<PopoverTrigger >
+							<Grid role="button" w="100%" templateColumns={collateralData.filter(x => x != null).join('% ')+"%"} h="2rem" borderRadius="8px" borderColor="#444" borderStyle="solid" borderWidth="3px" overflow="hidden">
+								{allReservesData?.map((token, index) =>
+									<Box bg={assetColor[token.symbol]} w="100%" h="100%" borderRadius="0" _hover={{ bg: assetColor[token.symbol], boxShadow: "xl", }} _active={{ bg: assetColor[token.symbol] }} _focus={{ boxShadow: "xl" }} d={(collateralComposition[index] != null) ? 'block' : 'none'} />
+								)}
+							</Grid>
+						</PopoverTrigger>
+						<PopoverContent bg="primary.300" border="2px solid" >
+							<PopoverBody bg="gray.700">
+								<VStack  m='auto' py='2rem' w="90%">{allReservesData?.map((token, index) =>
+									(collateralComposition[index] != null) ?
+										<Flex alignItems="center" justifyContent="space-between" w='100%'>
+											<Box bg={assetColor[token.symbol]} boxSize="1em" minW='1em' minH='1em' borderRadius="1em"/>
+											<Text ml="1em" width="50%">  {token.symbol}</Text>
+											<Text ml="1em">  {collateralData[index]+'%'}</Text>
+										</Flex>
+										: <Text></Text>
+								)}</VStack>
+							</PopoverBody>
+						</PopoverContent>
+					</Popover>
+				</Flex>
+			</Flex>
+		</VStack >
+	);
+};
+
+export default WithdrawDash;
```

Comment:
Consider default-exports problematic.
```suggestion
export default WithdrawDash;
```

Reply:
Fixed also on DepositDash